### PR TITLE
feat: page home search explore 1

### PR DESCRIPTION
**Pull request by 2nthony**
#188

### CODE REVIEW

File: site/src/components/home/explore/dropdown.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        <DropdownItemContentDescription>
+          {description}
+        </DropdownItemContentDescription>
+      </DropdownItemContent>
+      <DropdownItemIndex>#{value}</DropdownItemIndex>
+    </DropdownLinkItem>
+  );
+}
+
+export default function ExploreDropdown({ hints, visible }) {
```
Comment (wliyongfeng) [inline site/src/components/home/explore/dropdown.js:89]: How about adding code here:
```javascript
if (!visible) {
  return null
}
```

File: site/src/components/home/explore/dropdown.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  if (!visible) {
+    return null;
+  }
+
+  return (
+    <Dropdown>
+      {hints.map((hint) => (
+        <DropdownGroup key={hint.type}>
```
Comment (YoshiyukiSakura) [inline site/src/components/home/explore/dropdown.js:97]: be careful with multiple hints with the same type
could happen once we add `asset` type

File: site/src/components/home/explore/index.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  const chain = useSelector(chainSelector);
+  const [term, setTerm] = useState("");
+  const [hints, setHints] = useState([]);
+  const [dropdownVisible, setDropdownVisible] = useState(false);
+
+  async function fetchHints(term) {
+    return api.fetch(homeSearchHints, { term }).then(({ result }) => {
+      const data = compatExploreDropdownHints(result);
+
+      if (data.length) {
```
Comment (YoshiyukiSakura) [inline site/src/components/home/explore/index.js:68]: might be replaced with 
``` javascript
if(!result?.length){
  return;
}

File: site/src/components/home/explore/index.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  function onBlur() {
+    setDropdownVisible(false);
+  }
+
+  return (
+    <div>
+      <Title>{chain} Explorer</Title>
+      <Wrapper style={{ gap: 16 }}>
+        <Input
+          placeholder={"Block / Address / Extrinsic / Asset /..."}
```
Comment (YoshiyukiSakura) [inline site/src/components/home/explore/index.js:110]: no `Asset` right now

File: site/src/components/home/explore/index.js (hunk 3 above)
Comment (2nthony) [reply site/src/components/home/explore/index.js:68]: `result` is an `object`

**Pull request merged**